### PR TITLE
Tests/visit all admin

### PR DESCRIPTION
**Pull request by billybonks**
Fixes 

https://github.com/HospitalRun/hospitalrun-frontend/issues/687
https://github.com/HospitalRun/hospitalrun-frontend/issues/688

**Changes proposed in this pull request:**
Just visits these two routes during acceptance tests to make sure translations are in place

cc @HospitalRun/core-maintainers

### CODE REVIEW

File: tests/acceptance/admin-test.js [hunk 1]
```diff
@@ -32,6 +32,29 @@ test('visiting /admin', function(assert) {
...
+      assert.equal(currentURL(), url);
+      assert.equal(find('.view-top-bar .view-current-title').text(), 'Load DB');
+    });
+  });
+});
+
+test('visiting /admin/query', function(assert) {
+  runWithPouchDump('admin', function() {
+    authenticateUser();
+    var url = '/admin/query'
```
Comment (jkleinsc) [inline tests/acceptance/admin-test.js]: Missing semicolon.

File: tests/acceptance/admin-test.js [hunk 2]
```diff
@@ -32,6 +32,29 @@ test('visiting /admin', function(assert) {
   });
 });
 
+test('visiting /admin/loaddb', function(assert) {
+  runWithPouchDump('admin', function() {
+    authenticateUser();
+    var url = '/admin/loaddb'
```
Comment (jkleinsc) [inline tests/acceptance/admin-test.js]: missing semicolon

File: tests/acceptance/admin-test.js (hunk 1 above)
Comment (Anisha1234) [inline tests/acceptance/admin-test.js]: @jkleinsc I would like to solve the above mentioned travis error.

**Comment by Anisha1234** [comment]
@billybonks  @jkleinsc I would like to solve the travis error

**Comment by billybonks** [comment]
@Anisha1234 hey, i think the only bit of code i would want to save is

https://github.com/HospitalRun/hospitalrun-frontend/pull/721/commits/4a5b01fbffae2def168721d0715b9bd8f5e051b5

the rest is pretty out of date, feel free to pull out the commit into your own branch and modify from there

**Comment by tangollama** [comment]
@Anisha1234 see @billybonks comment. I'm intending to close this PR, but feel free to follow his instruction and submit a new PR.

**Pull request closed without merging**